.travis.yml

```
@@ -3,10 +3,7 @@ os:
   - osx
   - linux
 julia:
-  - 1.0
-  - 1.1
-  - 1.2
-  - 1.4
+  - 1.5
```

It still worth checking if this package works nicely with the latest LTS version (1.0.5), while `1` means the latest stable version (1.5.3)

```suggestion
  - 1.0
  - 1
```

---

ok, but 1.5 is soooo much better i can't imagine why anyone would use 1.0.x !

---

It is, but as a package maintainer, we should respect LTS and backward-compatibility if we can.